Formula/boost-python3.rb

```diff
@@ -0,0 +1,90 @@
+class BoostPython3 < Formula
+  desc "C++ library for C++/Python3 interoperability"
+  homepage "https://www.boost.org/"
+  url "https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2"
+  sha256 "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9"
+  head "https://github.com/boostorg/boost.git"
+
+  bottle do
```

Comment:
This is a new formula so it shouldn't have a bottle block yet

Reply:
All done